src/SourceBuild/content/prep.sh

```diff
@@ -10,11 +10,17 @@ usage() {
     echo "  Prepares the environment to be built by downloading Private.SourceBuilt.Artifacts.*.tar.gz and"
     echo "  installing the version of dotnet referenced in global.json"
     echo "options:"
-    echo "  --no-bootstrap Don't replace portable packages in the download source-built artifacts"
+    echo "  --no-bootstrap    Don't replace portable packages in the download source-built artifacts"
+    echo "  --no-artifacts    Exclude the download of the previously source-built artifacts archive."
+    echo "  --no-prebuilts    Exclude the download of the prebuilts archive."
+    echo "  --no-sdk          Exclude the download of the .NET SDK."
```

Comment:
There is a potential side effects of specifying no-sdk.  Specifically the bootstrap logic will fail if there is no SDK.  It feels like it would be appropriate to check for this condition and error out rather than attempting to bootstrap as I suspect this will not provide a meaningful UX.  

When checking for this potential error condition, you may choose to provide some leniency.  By that I mean you could support --no-sdk and perform the bootstrap if the caller has setup `$SCRIPT_ROOT/.dotnet` themselves.